tests/test_metric_regrouping.js

```diff
@@ -276,16 +276,18 @@ test('bundling', function(t) {
       jsonArray = http_client.buildPayloadsUnsafe('11111111', 22222222, gauges, counters, timers, sets, 15000, 200),
       actualMetricCount = 0;
 
-  // it should have been broken up into 12 bundles.
-  t.equal(14, jsonArray.length, 'The payload was broken into 14 bundles.');
+  // it should have been broken up into 14 bundles.
```

Comment:
I'd prefer this comment not even be here since it is redundant with the test's message and the test's message is slightly more correct and clearer.